docs/src/dev/env.md

```diff
@@ -17,12 +17,19 @@ build *without* the feature enabled, do `cargo build -p ykcapi`).
 
 ### `YKB_YKLLVM_BIN_DIR`
 
-Under normal circumstances, yk builds a copy of its LLVM fork "ykllvm" and uses
-it to build interpreters. You can use your own ykllvm build by specifying the
-directory where the executables (e.g. `clang`, `llvm-config`, and so on) are
-stored with `YKB_YKLLVM_BIN_DIR`. yk does not check your installation for
-compatibility: it is your responsibility to ensure that your ykllvm build
-matches that expected by yk.
+Under normal circumstances, yk builds a copy of its LLVM fork "ykllvm", which
+it also uses it to build interpreters (via the compilers `yk-config`). You can
```

Comment:
s/compilers/compiler's?

Reply:
Oops, that bracket bit doesn't parse, full stop! Fixed in 6f96334.